src/app/infobox/infobox.component.ts

```diff
@@ -41,8 +49,14 @@ export class InfoboxComponent implements OnInit {
         }
 
     });
+  }
 
+  if (synthesisEnable = true) {
```

Comment:
Done @nikhilrayaprolu. I tried to pass `this.results` the value in the function. But it utters 'object object'